programs/castle-vault/src/adapters/jet.rs

```diff
@@ -180,3 +181,89 @@ impl ReserveAccessor for Reserve {
         Ok(reserve)
     }
 }
+
+#[derive(Accounts)]
+pub struct JetRefresher<'info> {
```

Comment:
Nit: can you change these account structs to be the same as the IX: `RefreshJet`